core/services/workflows/delegate.go

```diff
@@ -67,8 +76,40 @@ func (d *Delegate) ServicesForSpec(ctx context.Context, spec job.Job) ([]job.Ser
 	return []job.ServiceCtx{engine}, nil
 }
 
-func NewDelegate(logger logger.Logger, registry types.CapabilitiesRegistry, legacyEVMChains legacyevm.LegacyChainContainer) *Delegate {
-	return &Delegate{logger: logger, registry: registry, legacyEVMChains: legacyEVMChains}
+func initializeDelayedExecution(myPeerID string) (delayedExecution, error) {
+	// TODO: source the below from the registry
+	workflowDONPeers := []string{
+		"p2p_12D3KooWF3dVeJ6YoT5HFnYhmwQWWMoEwVFzJQ5kKCMX3ZityxMC",
```

Comment:
Drop the "p2p_" prefix here. I saw you were getting some errors about it in staging.

Reply:
Hmmmm... I added it because myPeerID had a p2p prefix added to it

Reply:
That shouldn't be the case any more, right? I don't think the internal types add a prefix anywhere (I hope).